src/components/PullRequestList/PullRequestListItem/index.js

```diff
@@ -1,4 +1,4 @@
-/* @flow */
+// TODO: fix flow for build property
 
```

Comment:
We can model, but I don't think it makes sense just for the sake of enabling flow here. Component if still not completed and depends on builds results API.